build/azure-pipelines/cli/cli-compile-and-publish.yml

```diff
@@ -8,26 +8,20 @@ parameters:
     default: {}
 
 steps:
-  - script: cargo build --release --target ${{ parameters.VSCODE_CLI_TARGET }} --bin=code-tunnel
+  - script: cargo build --release --target ${{ parameters.VSCODE_CLI_TARGET }} --bin=code
     displayName: Compile ${{ parameters.VSCODE_CLI_TARGET }}
     workingDirectory: $(Build.SourcesDirectory)/cli
     env:
-      VSCODE_CLI_VERSION: $(VSCODE_CLI_VERSION)
-      VSCODE_CLI_REMOTE_LICENSE_TEXT: $(VSCODE_CLI_REMOTE_LICENSE_TEXT)
-      VSCODE_CLI_REMOTE_LICENSE_PROMPT: $(VSCODE_CLI_REMOTE_LICENSE_PROMPT)
-      VSCODE_CLI_ASSET_NAME: ${{ parameters.VSCODE_CLI_ARTIFACT }}
-      VSCODE_CLI_AI_KEY: $(VSCODE_CLI_AI_KEY)
-      VSCODE_CLI_AI_ENDPOINT: $(VSCODE_CLI_AI_ENDPOINT)
       ${{ each pair in parameters.VSCODE_CLI_ENV }}:
         ${{ pair.key }}: ${{ pair.value }}
 
   - ${{ if or(contains(parameters.VSCODE_CLI_TARGET, '-windows-'), contains(parameters.VSCODE_CLI_TARGET, '-darwin')) }}:
     - task: ArchiveFiles@2
       inputs:
         ${{ if contains(parameters.VSCODE_CLI_TARGET, '-windows-') }}:
-          rootFolderOrFile: $(Build.SourcesDirectory)/cli/target/${{ parameters.VSCODE_CLI_TARGET }}/release/code-tunnel.exe
+          rootFolderOrFile: $(Build.SourcesDirectory)/cli/target/${{ parameters.VSCODE_CLI_TARGET }}/release/code.exe
```

Comment:
These changes (and most of the ones in the build script below) are related to renaming the standalone CLI from `code-tunnel` to `code`. It still gets bundled as `code-tunnel` in the desktop builds, but users who download the VS Code CLI should just get `code`.